src/loki/Counters.h

```diff
@@ -0,0 +1,53 @@
+/* Copyright (C) 2019 European Spallation Source, ERIC. See LICENSE file */
+//===----------------------------------------------------------------------===//
+///
+/// \file
+///
+/// \brief Holds application counters
+///
+//===----------------------------------------------------------------------===//
+
+#pragma once
+
+#include <cinttypes>
+
+struct Counters {
+  // Input Counters - accessed in input thread
+  int64_t RxPackets;
+  int64_t RxBytes;
+  int64_t FifoPushErrors;
+  int64_t InputIdle;
+  int64_t PaddingFor64ByteAlignment[4]; // cppcheck-suppress unusedStructMember
+
+  // Processing Counters - accessed in processing thread
+  int64_t FifoSeqErrors;
```

Comment:
Is there a reason for the type int64_t ? In principle the error counters should only have positive values, so uint64_t would be more apt.

Reply:
Yes there is, probably :-)

I initially thought that the stats could represent any integers. In practice they are 
all counters. But I would not want to changes this now. I even think Jonas uses negative values for a specific case I have long forgotten.